workshops/how_we_enable/03-Strategy.md

```diff
@@ -1,12 +1,51 @@
+Automating all services and making them consumable via APIs is the standard way of doing this in today's IT. Most people start with a command line tool, and build it into a pipeline or pass around cookbooks or playbooks, which is a great way to start, however someone needs to know where these are. A common practice is using a centralized tool like Ansible Tower to make all of these playbooks self service by exposing them as APIs.
+
+## Cloud Native Development
+
+All new application development should follow Cloud Native Standards:
+
+| Standard                | Description                                                      |
+|-------------------------|------------------------------------------------------------------|
+| **Codebase**            | One codebase tracked in revision control, many deploys           |
+| **Dependencies**        | Explicitly declare and isolate dependencies                      |
+| **Config**              | Store config in the environment                                  |
+| **Backing services**    | Treat backing services as attached resources                     |
+| **Build, release, run** | Strictly separate build and run stages                           |
+| **Processes**           | Execute the app as one or more stateless processes               |
+| **Port binding**        | Export services via port binding                                 |
+| **Concurrency**         | Scale out via the process model                                  |
+| **Disposability**       | Maximize robustness with fast startup and graceful shutdown      |
+| **Dev/prod parity**     | Keep development, staging, and production as similar as possible |
+| **Logs**                | Treat logs as event streams                                      |
+| **Admin processes**     | Run admin/management tasks as one-off processes                  |
+
+_Source:_ [12factor.net](https://www.12factor.net)
+
+## CI/CD
+
+Continuous integration and continuous delivery/deployment is the defacto standard for delivering applications in a cloud native way.  Having a standardized pipeline that everyone uses to build, test, operate, measure, monitor, secure and deploy applications at the enterprise level helps everyone to follow the same standards for quality application delivery.
+
+## Monitoring/Observability
+
+Much like CI/CD having consistent cross functional standards for logging (preferably as streams) monitoring, tracing and utilization is a best practice to have.
+
+## Security
+
+Security can be made easier for developers when you solidify a standard for them to use and implement into their CI/CD pipelines. This includes but is not limited to the implementation of static code analysis as well as running code analysis tools. It is also a best practice to use a trusted source for libraries that is regularly scanned (ie not pulling libraries directly from the internet pypi, npm etc)
```

Comment:
Add links to pypi and npm

Reply:
fixed in 76efbc72e9c9b6c16795e88904f82f94fe94352c